Sources/Widgets/Widgets3D/LineWidget/state.js

```diff
@@ -15,7 +15,7 @@ export default function generateState() {
     })
     .addStateFromMixin({
       labels: ['handle1'],
-      mixins: ['origin', 'color', 'scale1', 'visible'],
+      mixins: ['origin', 'color', 'scale1', 'visible', 'manipulator'],
```

Comment:
what is it for ?

Reply:
not sure, but manipulators are used in OFSEP (behavior similar to intensityWidget). Should i remove it?